### PR TITLE
Website Landing Page

### PR DESCRIPTION
**Pull request by akestner**
**Issue, if available:**
N/A

**Description of changes:**
This PR updates the website's landing page to utilize the project's branding, speak more comprehensively to its benefits, and provide a high level technical overview.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by geoffcline** [comment]
homepage looks great so far!

File: website/assets/scss/_variables_project.scss [hunk 1]
```diff
@@ -3,3 +3,75 @@
 Add styles or override variables from the theme here.
 
 */
+
+// Color palatte (AWS branding)
```
Comment (ellistarn) [inline website/assets/scss/_variables_project.scss:7]: This doesn't feel super neutral to me.

File: website/content/en/_index.html [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    <a class="btn btn-lg btn-dark mb-3 px-5 font-weight-bold" href="{{< relref "docs/getting-started" >}}">
+      Get Started
+    </a>
+  </div>
+</div>
+{{< /blocks/cover >}}
+
+{{< blocks/lead color="grey-100" height="med">}}
+  <h2 class="mx-auto mt-5 col-sm-9 font-weight-bolder">
+    Karpenter simplifies Kubernetes infrastructure with the right nodes at just the right time.
```
Comment (ellistarn) [inline website/content/en/_index.html:23]:
```suggestion
    Karpenter simplifies Kubernetes infrastructure with the right nodes at the right time.
```

File: website/assets/scss/_variables_project.scss (hunk 1 above)
Comment (akestner) [reply website/assets/scss/_variables_project.scss:7]: Had to start somewhere, to be fair this is no longer entirely accurate in that we primarily use non-AWS colors. I can update this to better reflect that

Comment (akestner) [reply website/assets/scss/_variables_project.scss]: Updated to include only necessary colors

**Review by geoffcline — APPROVED** [review]

File: website/layouts/partials/footer.html [hunk 3]
```diff
@@ -0,0 +1,44 @@
+{{ $links := .Site.Params.links }}
+<footer class="bg-dark py-5 row d-print-none">
+  <div class="container-fluid mx-sm-5">
+    <div class="row">
+      <p class="mx-auto col-sm-3 text-center text-light">
+        Built with ❤️ at <a class="text-light" href="https://aws.amazon.com/">AWS</a>
```
Comment (geoffcline) [inline website/layouts/partials/footer.html:6]: Could we move this from a universal footer to an "about us" type page? where that page would include info on contributing to the project?

Comment (geoffcline) [inline website/layouts/partials/footer.html:6]: Could we move this from a universal footer to an "about us" type page? where that page would include info on contributing to the project?

**Review by geoffcline — CHANGES_REQUESTED** [review]

**Comment by geoffcline** [comment]
Overall, excellent work B)

File: website/layouts/partials/footer.html (hunk 3 above)
Comment (geoffcline) [reply website/layouts/partials/footer.html:6]: related to new issue #579

**Review by ellistarn — APPROVED** [review]

**Comment by ellistarn** [comment]
Feel free to merge whenever!

**Pull request merged**